### PR TITLE
Added my photo to student section

### PR DESCRIPTION
**Pull request by shreyankt99**
Uodated student.yml and added my img
https://shreyankt99.github.io/gci17.fossasiashreynkt.org/.

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -297,10 +297,10 @@
   github: Akharote1
   image: aditya_kharote.jpg
   twitter: Akharote1
-
-- name: Dai Tianle
-  country: Singapore
-  github: ddx-510
-  image: ddx.jpg
-  twitter: ddx2000
-
```
Comment (nikit19) [inline _data/students.yml:306]: Don't delete somebody's code. You can write after them

**Review by nikit19 — CHANGES_REQUESTED** [review]
Looks good just make this change

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Squash your commits

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Follow @nikit19 's review, resolve conflicts and squash all your commits.

**Review by isuruAb — COMMENTED** [review]
please squash your commits and resolve conflicts

**Review by ParthS007 — COMMENTED** [review]
@shreyankt99 Closing this PR due to many conflicts and inactivity.Please read `README.MD` on how to add a student and make a PR.Please ask on FOSSASIA Gitter channel if you face any difficulty 👍

**Pull request closed without merging**